apps/scan/frontend/src/components/layout.tsx

```diff
@@ -30,11 +30,11 @@ export interface ScreenProps {
   hideBallotCount?: boolean;
   centerContent?: boolean;
   children: React.ReactNode;
-  isLiveMode?: boolean;
   infoBarMode?: InfoBarMode;
   hideInfoBar?: boolean;
   padded?: boolean;
   title?: React.ReactNode;
+  showTestModeBanner: boolean;
```

Comment:
- changing the name, because sometimes it will be test mode but we don't want to show the banner
- making required, so when using the component we have to think about whether the test mode banner should show or not